/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-01.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-01.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
01
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-02.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-02.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
02
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-03.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-03.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
03
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-04.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-04.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
04
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-05.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-05.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
05
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-06.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-06.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
06
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-07.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-07.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
07
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-08.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-08.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
08
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-09.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-09.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
09
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-10.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-10.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
10
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-11.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-11.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
11
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-12.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-12.monthly.mean.nc
Traceback (most recent call last):
  File "./plot_cru.py", line 3, in <module>
    import makeplot_cru
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 89
    labels,subtitle,titlem,pathout,outnammem,outformat,\
    ^
IndentationError: unexpected indent
12
1990-1999
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-01.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-01.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
01
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-02.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-02.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
02
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-03.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-03.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
03
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-04.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-04.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
04
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-05.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-05.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
05
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-06.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-06.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
06
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-07.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-07.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
07
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-08.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-08.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
08
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-09.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-09.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
09
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-10.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-10.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
10
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-11.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-11.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
11
/nfs/hydro6/raid/nijssen/rasm/r33RBVIC60/lnd/hist/r33RBVIC60.vic.ha.1990-1999-12.monthly.mean.nc
/raid/muxiao/crudata/cru_shape/cru_ts_3_10_01.pre.1990-1999-12.monthly.mean.nc

 ok
the output directory is: /raid/muxiao/rasmfig/cru1
Traceback (most recent call last):
  File "./plot_cru.py", line 22, in <module>
    main()
  File "./plot_cru.py", line 14, in main
    makeplot_cru.main(a[i], b[i], c[i], d[i])
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/makeplot_cru.py", line 94, in main
    projection_parameters)
  File "/usr1/muxiao/working_rasm/9_11/plot_cru/layout.py", line 18, in create_plot
    fig = plt.figure()
  File "/usr/local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 343, in figure
    **kwargs)
  File "/usr/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 80, in new_figure_manager
    window = Tk.Tk()
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 1685, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
12
1990-1999
